source/opt/aggressive_dead_code_elim_pass.cpp

```diff
@@ -123,40 +129,193 @@ void AggressiveDCEPass::ProcessLoad(uint32_t varId) {
   live_local_vars_.insert(varId);
 }
 
+uint32_t AggressiveDCEPass::MergeBlockIdIfAny(
```

Comment:
There are copies of this already in InlinePass, LocalMultiStoreElimPass, DeadBranchElimPass, CommonUniformElimPass.  Let's not add a fifth copy.

Reply:
Yes. Sorry. I am just trying to get this code out as quickly as possible. I know of two Vulkan development groups that are blocked waiting for this code.

The LocalMultiStoreElimPass version is different than the others (it includes all blocks, not just unreachable ones). That is what is holding up merging these all together. That and I still need to move CommonUniformElimPass under MemPass. I will do this under a separate PR.